internal/auth/aes_encryptor.go

```diff
@@ -41,18 +41,18 @@ func (ae *AESEncryptor) Encrypt(message string) (string, error) {
 
 	block, err := aes.NewCipher(ae.Key)
 	if err != nil {
-		return "", fmt.Errorf("Unable to create cipher: %v", err)
+		return "", fmt.Errorf("unable to create cipher: %v", err)
```

Comment:
Hmm, OK but why downcasing? It will be concatenated with something else?
EDIT: I understand this was part of a go-linter-related change, according to the commit title. What was the issue raised?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings